### PR TITLE
Store tenant creator id in realm when creation tenants by organization creation 

### PR DESCRIPTION
**Pull request by AnuradhaSK**
## Purpose
part of fix Fix https://github.com/wso2/product-is/issues/14001

### Related PRs

- [x] Bump kernel version after merging https://github.com/wso2/carbon-kernel/pull/3403

### CODE REVIEW

File: components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImpl.java [hunk 1]
```diff
@@ -842,16 +842,12 @@ private void createTenant(String domain, String orgCreatorID, String orgCreatorN
...
         tenant.setDomain(domain);
-        /*
-        Set the creator's UUID as the tenant admin name because tenant data model doesn't store the admin uuid,
-        and it is required when creating the org-user association.
-         */
-        tenant.setAdminName(orgCreatorID);
+        tenant.setAdminName(orgCreatorName);
         tenant.setAdminUserId(orgCreatorID);
         tenant.setEmail("dummyadmin@email.com");
         // set the password as domain for now to avoid findbugs detecting it as a hardcoded value.
```
Comment (madurangasiriwardena) [inline components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImpl.java:853]: If we avoid setting anything for email and password, will it break anything?

**Review by madurangasiriwardena — APPROVED** [review]

File: components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImpl.java (hunk 1 above)
Comment (AnuradhaSK) [reply components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImpl.java:853]: There is an email non-empty validation https://github.com/wso2/carbon-multitenancy/blob/5a9360e09fcc4b0dfa3e036626c99b3d598c1be4/components/tenant-mgt/org.wso2.carbon.tenant.mgt/src/main/java/org/wso2/carbon/tenant/mgt/services/TenantMgtImpl.java#L396

Comment (AnuradhaSK) [reply components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImpl.java]: password is not mandatory. So I'll remove that

**Comment by AnuradhaSK** [comment]
close this PR with https://github.com/wso2-extensions/identity-organization-management-core/pull/17

**Pull request closed without merging**